### DISCUSSION TITLE
Compiled clean and works for kali 6.8 on a Raspberry PI CM4

### DISCUSSION THREAD

**Discussion by z00mantwo** [morrownr/8814au#146] [General]
Thank you! Compiled clean and works for kali 6.8 on a Raspberry PI CM4. This is on a Clockwork uConsole CM4.

https://www.clockworkpi.com/uconsole

**Comment by morrownr** [comment]
Good to hear. It is not the greatest driver for Kali so if you decide to look for something better at some point:

https://github.com/morrownr/USB-WiFi

Menu item 2 is the Plug and Play List then under it is a list of the adapters that are very good for Kali.

**Comment by z00mantwo** [comment]
Great list! Wish I had seen it sooner.
